src/components/menus/line-menu.js

```diff
@@ -86,7 +85,8 @@ const withLineMenu =
         const getLineDescriptor = useCallback(
             (voltageLevelId) => {
                 return displayUseName
-                    ? network.getVoltageLevel(voltageLevelId).name
+                    ? network.getVoltageLevel(voltageLevelId).name ??
```

Comment:
You don't want to keep safe with `?.name` ?

Reply:
yes, done